backend/tournesol/serializers/contributor_recommendations.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.serializers import ModelSerializer
+
+from tournesol.models import ContributorRating, ContributorRatingCriteriaScore
+from tournesol.serializers.poll import RecommendationSerializer
+
+
+class ContributorRecommendationsScore(ModelSerializer):
+    """ Outputs the recommending contributor's criteria scores for the given entity """
+    class Meta:
+        model = ContributorRatingCriteriaScore
+        fields = ["criteria", "score"]
+
+
+class ContributorRecommendationsRating(ModelSerializer):
+    criteria_scores = ContributorRecommendationsScore(many=True)
+
+    class Meta:
+        model = ContributorRating
+        fields = ["is_public", "criteria_scores"]
+
+
+class ContributorRecommendationsSerializer(RecommendationSerializer):
+    """ Format the output of the contributor_recommendations views """
```

Comment:
As the name of the class is concise and explicit, you can choose to not add a docstring here.